src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/MaintainSupplierPricingController.kt

```diff
@@ -28,6 +29,7 @@ import javax.validation.constraints.NotNull
   HtmlController.DROP_OFF_ATTRIBUTE,
   HtmlController.DATE_ATTRIBUTE
 )
+@PreAuthorize("hasRole('PECS_MAINTAIN_PRICE')")
```

Comment:
Using class level as all of the controller methods here required role this particular role enforcement.